.travis.yml

```diff
@@ -65,7 +66,7 @@ stages:
   - name: Tests - phase 2
     if: type != cron
   - name: cron
-    if: type = cron
+    if: type = cron OR (type != pull_request AND branch != master)
```

Comment:
https://docs.travis-ci.com/user/conditions-v1 says the possible `type` are `push`, `pull_request`, `api`, `cron`. So these tests will now additionally run for nonmaster branches if the event type is `push` or `api`. I guess `api` is for manually-triggered tests on branches. What's `push` for?

Reply:
I am guessing but it must be triggered for any commit that is `pushed` to the branch. I could not explicit documentation but that definition seems most plausible.

Reply:
Ok, let's try this and see if it does what we want.